test/public.js

```diff
@@ -107,3 +107,33 @@ describe('Plaid.getInstitutions', function() {
   });
 
 });
+
+describe('Plaid.searchInstitutions', function() {
+
+  it('returns a single institution given an "id"', function(done) {
+    Plaid.searchInstitutions({
+      id: 'bofa',
+    }, Plaid.environments.tartan, function(err, res) {
+      eq(err, null);
+
+      eq(res.id, 'bofa');
+      assert(R.is(Object, res));
+
+      done();
+    });
+  });
+
+  it('returns a list of institutions given a "p" and "q"', function(done) {
+    Plaid.searchInstitutions({
+      product: 'connect',
+      query: 'suntrust',
+    }, Plaid.environments.tartan, function(err, res) {
+      eq(err, null);
+
+      assert(R.is(Array, res));
```

Comment:
Better to write `eq(R.type(res), 'Array');` so we'll get a descriptive message if the assertion fails. :)